Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace.package]
 version = "1.0.8"
 edition = "2021"
-rust-version = "1.81"
+rust-version = "1.81.0"
```

Comment:
Inside of the flake, I am parsing the rust-version from the `Cargo.toml` like so
```nix
      rustVersion = cargoTOML.workspace.package.rust-version;
      rustPkg = pkgs.rust-bin.stable."${rustVersion}".default.override {
        extensions = [ "rust-src" "rust-analyzer" "clippy" "rustfmt" ];
      };
```
in order to determine which version of Rust is needed to in the build steps and development environment.
But since 1.81 is not an exact match with the Rust release version number the rust-overlay input cannot find a suitable rust version.
By using the release version written [here](https://doc.rust-lang.org/stable/releases.html) it ensures that the flake doesn't need to be manually updated with a Rust version moving forward.